src/Component/FdaApprrove/Fda.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+
+export function Fda() {
+  const [fda, setFda] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get( `https://vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com/api/vaccines/get-fda-approved-vaccines`,
+        {
+             headers: {
+            'X-RapidAPI-Key': '8c0c5645f4msh17b4e89de7b874ap10eb20jsn3e95f4dad049',
+            'X-RapidAPI-Host': 'vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com'
+          },
+        }
+      )
+      .then(res => {
+        console.log(res.data);
```

Comment:
remove/comment console statements before pushing

Reply:
It is still not fixed

Reply:
removed